src/AuthenticationService.php

```diff
@@ -151,17 +155,17 @@ public function loadAuthenticator($name, array $config = [])
      * @param array $config Identifier configuration.
      * @return \Authentication\Identifier\IdentifierInterface Identifier instance
      */
-    public function loadIdentifier($name, array $config = [])
+    public function loadIdentifier(string $name, array $config = []): IdentifierInterface
     {
         return $this->identifiers()->load($name, $config);
     }
 
     /**
-     * {@inheritDoc}
+     * @inheritdoc
```

Comment:
wrong one, thats not the partial overwrite

Reply:
Casualty of search - replace :)